.circleci/config.yml

```diff
@@ -274,7 +274,7 @@ jobs:
           command: |
             just install
             forge --version
-            forge build --deny-warnings
+            forge build --deny-warnings --ignored-error-codes=5740 --ignored-error-codes=5574
```

Comment:
The bytecode limit only applies to src contracts that we intended to deploy, it does not apply to script or test contracts. We should instead make sure the corresponding contracts are marked as scripts or tests accordingly, which I believe forge does by checking for the `*.t.sol` and `*.s.sol` extensions. All of our templates and tasks should probably have `*.s.sol` since they are invoked via `forge script`?